packages/generator/src/edmx-to-vdm/edmx-to-vdm-util.ts

```diff
@@ -38,7 +54,11 @@ export function isV2Metadata(metadata: EdmxMetadata): boolean {
   return metadata.oDataVersion === 'v2';
 }
 
-export function isComplexType(type: string): boolean {
+export function isComplexType(typeName: string): boolean {
+  let type = typeName;
+  if (isCollectionType(typeName)) {
```

Comment:
I agree. I think this collection checking was not used because in the collection case I was doing the parseCollectionTypeName also before. So I removed it. Test stayed green.